cibuildwheel/__main__.py

```diff
@@ -136,11 +138,18 @@ def main():
     )
 
     if platform == 'linux':
-        manylinux1_x86_64_image = os.environ.get('CIBW_MANYLINUX1_X86_64_IMAGE', None)
-        manylinux1_i686_image = os.environ.get('CIBW_MANYLINUX1_I686_IMAGE', None)
+        # TODO Add documentation on CIBW_ENVIRONMENT and possibility of AUDITWHEEL_PLAT if not defined by custom manylinux image
+
+        manylinux_x86_64_image = os.environ.get('CIBW_MANYLINUX_X86_64_IMAGE', 'manylinux2010')
+        manylinux_i686_image = os.environ.get('CIBW_MANYLINUX_I686_IMAGE', '')
```

Comment:
hmmm.... not sure how I feel about this... this means that CIBW_BUILD can specify a i686 wheel, but it won't get built because there's no image specified. I think that's a bit confusing. CIBW_BUILD should specify which wheels are built.

I'd prefer this to specify the old manylinux1, and not allow this option to be empty.

Reply:
That's a good point. I hadn't realized that, actually. :-/
The problem is that there was a strong suggestion in pypa/manylinux#179 to discourage people from building on the `manylinux1` image, given the potential security issues in the old CentOS 5 image. And if not now, at some point `manylinux1` will be deprecated (and a newer 32-bit docker image might not become available).

Any thoughts on that, @joerick? Should we make `CIBW_SKIP="manylinux-i686"` the default? That doesn't feel too right either, somehow :-/

Reply:
On my library I decided to skip 32-bit Linux because <1% of Linux machines are i686. I figured it was not worth the energy and CO2 to have a server build binaries no one will ever use. Two years ago, I decided if just one person asked for them, I would make it available. 

Last year, someone opened an issue that amounted to failing to compile 32-bit from source because there was no manylinux1 wheel. I explained this, and the person said they selected a 32-bit virtual machine by mistake. They closed the issue. To date, no one has ever wanted i686. This library has on the order of 10k users including some enterprise ones.

I think x64 only is a reasonable default for Linux. (32-bit Windows is another matter entirely - there are still lots of 32-bit users.) But maybe it should be an explicit opt-in?

Reply:
Thanks for the perspective @jbarlow83! I wasn't aware of how rare that variant was. I might try to get some stats out of PyPI to back that up. 

As a mechanism to deselect manylinux-i686 by default, it should be something that doesn't surprise the user. The trouble with a default on CIBW_SKIP is that  setting that value (e.g. to skip a win32 build) would accidentally enable i686. 

I'm wondering about a default value for CIBW_BUILD - `*-manylinux_x86_64 *-win32 *-macosx_10_6_intel ...`, since I'm guessing CIBW_BUILD is less commonly used than CIBW_SKIP, and when it is used, it's specifically targeting something.

Reply:
See https://github.com/joerick/cibuildwheel/pull/155#issuecomment-529647114 - I don't think we can drop i686 yet!